### PR TITLE
Rename library and module package

### PR DESCRIPTION
**Pull request by zainfikrih**
- rename library
- rename package app
- rename package module

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 29
     buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "com.bcc.opensource.gridmenu"
+        applicationId "com.filkom.bcc.gridmenu"
```
Comment (adityapp) [inline app/build.gradle]: change to "com.bcc.gridmenu"

**Review by adityapp — CHANGES_REQUESTED** [review]
please change package name to "com.bcc.gridmenu"

**Review by adityapp — APPROVED** [review]

File: app/build.gradle [hunk 2]
```diff
@@ -23,7 +23,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation project(path: ':squaremenu')
```
Comment (gifff) [inline app/build.gradle]: @adityapp is it better to use `gridmenu` for the library name and change the sample app name to something else?

File: app/build.gradle [hunk 3]
```diff
@@ -23,7 +23,7 @@ dependencies {
...
+    implementation project(path: ':gridmenulibrary')
```
Comment (gifff) [inline app/build.gradle]: @adityapp is it better to use `gridmenu` for the library name and change the sample app name to something else?

**Review by adityapp — APPROVED** [review]

**Review by gifff — APPROVED** [review]

**Pull request merged**